dkms/mt76-mt7925-dkms.spec

```
@@ -0,0 +1,38 @@
+%define module_name mt76-mt7925
```

RuboCop (1.82.1)

[fatal] unterminated string meets end of file
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)